docs/source/en/api/models/transformer2d.mdx

```diff
@@ -0,0 +1,29 @@
+# Transformer2D
+
+A Transformer model for image-like data. [`Transformer2DModel`] accepts discrete (classes of vector embeddings) or continuous (actual embeddings) inputs.
```

Comment:
Do we know anything concrete about its origin? Might be good to include that. 

@patrickvonplaten

Reply:
It comes from the CompVis repo, but really it's just a copy of the orignial ViT paper from Google.